### PR TITLE
Major code refactoring

### PR DESCRIPTION
**Pull request by sandrich**
- Refactored code into a NodeMonitor class
- Read rpcuser and rpcpassword from defi.conf
- Add --defi-conf parameter
- Removed --rpcuser and --rpcpassword parameter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ For a closed look in it's functionality there's a [detailled documentation](http
 - Install pip3 (pip from python v3. Some operating systems just name it ```pip```)
 - Run ```pip3 install masterhode-health```
 
+Make sure you set ```rpcuser=xxx``` and ```rpcpassword=xxx``` in your defi.conf
+
```
Comment (adrian-schnell) [inline README.md:18]:
```suggestion
You can choose the username and password by yourself. 

**Hint:** restart your node after updating your defi.conf
```

File: README.md [hunk 2]
```diff
@@ -62,32 +64,29 @@ To keep it simple, the following examples do not contain the relative path like
...
+                        Path to your defi.conf. Default: ~/.defi/defi.conf
   --api-key API_KEY     API Key
   --version             Returns masternode-health version
 ```
 
 You can manually run it with
 
 ```
-$ masternode-health --rpcuser rpc-username --rpcpassword rpc-password --defi-path /home/system-user/.defi --verbose --api-key=your-api-key
+$ masternode-health --verbose --api-key=your-api-key
```
Comment (adrian-schnell) [inline README.md:89]:
```suggestion
$ masternode-health --api-key=your-api-key
```

Comment (sandrich) [reply README.md:89]: hmm without --verbose it will not show the output below.

File: README.md (hunk 1 above)
Comment (sandrich) [reply README.md:18]: fine with me

**Review by adrian-schnell — APPROVED** [review]

**Pull request merged**